plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java

```diff
@@ -75,6 +86,44 @@
  */
 final class TikaImpl {
 
+    static {
+        /*
+         * Stop PDFBox from consulting the OS for fonts at all, use classpath instead with dummy fonts because font
+         * does not matter for ingestion
+         */
+        FontMappers.set(new FontMapper() {
+            @Override
+            public FontMapping<TrueTypeFont> getTrueTypeFont(String baseFont, PDFontDescriptor fd) {
+                try (InputStream in = TikaImpl.class.getResourceAsStream("/fonts/Roboto-Regular.ttf")) {
```

Comment:
Why do we need a "real" font here, if `return new FontMapping<>(null, true);` as fallback have the same effect?

Reply:
Let me double check this, if the test pass then we can remove the Roboto fonts.

Reply:
Just returning `return new FontMapping<>(ttf, true);` breaks the tests and PDFbox expects some font even though it does match the PDF’s look.
Error:
```
Caused by:
        java.lang.NullPointerException: Cannot invoke "org.apache.fontbox.FontBoxFont.getName()" because "this.genericFont" is null
 ```